lib/config.js

```diff
@@ -63,7 +61,7 @@ config.mongodb.writeOptions = {
 
 config.mongodb.requirements = {};
 // server version requirement with server-style string
-config.mongodb.requirements.serverVersion = '>=4.2';
+config.mongodb.requirements.serverVersion = '>=4.4';
```

Comment:
This is a breaking change, isn't it? Do we need to do this for this release?

Reply:
Ok reverted here: https://github.com/digitalbazaar/bedrock-mongodb/pull/89/commits/65d4986a44b2985a5f3c22ba2a3a67833be5c6e1